rever.xsh

```diff
@@ -1,10 +1,14 @@
+$PROJECT = 'regolith'
 $REVER_DIR = 'rever-tmp'
```

Comment:
I think that this line can / should be removed.  we can use the default rever dir here